lib/propcheck.ex

```diff
@@ -663,6 +663,20 @@ defmodule PropCheck do
         ...>   end)
         true
 
+    You also can refer variables declared in var scope with `^` until they
+    not form cycle or mix with var with same name from outer scope.
+
+        iex> use PropCheck
+        iex> nondec =
```

Comment:
`nondec` == `non_decreasing`? If yes, then change it please to make it easier to grasp your example.